CMakeLists.txt

```diff
@@ -30,12 +30,9 @@ cmake_print_variables(CMAKE_STAGING_PREFIX)
 set(DEPEND_INSTALL_DIR "$ENV{DEPEND_INSTALL}" CACHE PATH
     "Dependencies install directory")
 
-set(OVS_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}" CACHE PATH
+set(OVS_INSTALL_DIR "$ENV{OVS_INSTALL}" CACHE PATH
```

Comment:
Is user expected to set OVS_INSTALL variable after this change?

Reply:
But make-all.sh script is setting the default for OVS_DIR to ovs/install if env var OVS_INSTALL is empty (https://github.com/ipdk-io/networking-recipe/blob/2dd83088a56ddd4fc66cdc3c22b127e42e947181/make-all.sh#L20). We don't do that in CMakeLists.txt. So if the user is building using cmake directly, they are expected to set OVS_INSTALL env variable?